python-backend/app/scheduled_jobs/ETL_jobs.py

```diff
@@ -0,0 +1,20 @@
+from app.extensions import db, sched
+from app.api.utils.apm import register_apm
+
+
+#the schedule of these jobs is set using server time (UTC)
+def _schedule_ETL_jobs(app):
+    app.apscheduler.add_job(func=_run_ETL, trigger='cron', id='ETL', hour=11, minute=0)
+
+
+@register_apm
+def _run_ETL():
+    with sched.app.app_context():
+        db.session.execute('select transfer_mine_information();')
```

Comment:
Just wondering how the naming conventions for these functions came to be

Reply:
If its the _ prefix your wondering about it's there because these are internal use functions.

Reply:
Oh i didn't mean the private _ prefix for python.
The function for postgres with the `transfer_` prefix. Just curious on that

Reply:
Oh, I just tried to keep the stored procedure names for the ETL as close to the original names of the files as I could.